common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -19,15 +22,38 @@
 from .xml_module import XmlDescriptor
 from pkg_resources import resource_string
 
+
 # Make '_' a no-op so we can scrape strings
 _ = lambda text: text
 
 
+ANY_CAPA_TYPE_VALUE = 'any'
+
+
 def enum(**enums):
     """ enum helper in lieu of enum34 """
     return type('Enum', (), enums)
 
 
+def _get_human_name(problem_class):
+    """
+    Get the human-friendly name for a problem type.
+    """
+    return getattr(problem_class, 'human_name', problem_class.__name__)
+
+
+def _get_capa_types():
+    """
+    Gets capa types tags and labels
+    """
+    capa_types = {tag: _get_human_name(registry.get_class_for_tag(tag)) for tag in registry.registered_tags()}
+
+    return [{'value': ANY_CAPA_TYPE_VALUE, 'display_name': _('Any Type')}] + sorted([
+        {'value': capa_type, 'display_name': caption}
+        for capa_type, caption in capa_types.items()
+    ], key=lambda item: item.get('display_name'))
```

Comment:
Hrm... I guess `_get_capa_types` is pretty specific to this search code. Nevermind.